app/src/main/java/com/realityexpander/tasky/core/domain/typeParcelers/ZonedDateTimeParceler.kt

```diff
@@ -0,0 +1,16 @@
+package com.realityexpander.tasky.core.domain.typeParcelers
+
+import android.os.Parcel
+import kotlinx.parcelize.Parceler
+import java.time.ZonedDateTime
+
+object ZonedDateTimeParceler : Parceler<ZonedDateTime> {
```

Comment:
Same here

Reply:
I dont think the Parcelize will convert the custom types of LocalDateTime and ZonedDateTime, so I thought I needed to use a custom @Parceler. 

I will attempt to use it without the custom parceler and see if the @Parcelize does this for me.